pkg/internal/aws/aws.go

```diff
@@ -102,12 +109,16 @@ func (a *AWS) readSecret(ctx context.Context, id, version string) (map[string][]
 	req := a.client.GetSecretValueRequest(input)
 	resp, err := req.Send(ctx)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("error getting secret value: %s", err.Error())
```

Comment:
Errors should be wrapped.
```suggestion
		return nil, fmt.Errorf("error getting secret value: %w", err)
```